client/state/sites/plans/selectors.js

```diff
@@ -30,3 +30,8 @@ export function hasDomainCredit( state, siteId ) {
 	}
```

Comment:
@gwwar it'd be nice to move this whole state tree up one level out of `sites` as we are working through it.